### PR TITLE
Get cron jobs and celery workers working

### PR DESCRIPTION
**Pull request by glogiotatidis**
A number of fixes required to get cron jobs running (using APScheduler) and Celery working.

This has been manually pushed to sumo-dev.frankfurt and appears to be working OK

### CODE REVIEW

File: kitsune/settings.py [hunk 1]
```diff
@@ -928,7 +924,11 @@ def JINJA_CONFIG():
 
 
 # Google Analytics settings.
-GA_KEY = config('GA_KEY', default='longkey')  # Google API client key
+# GA_KEY is expected b64 encoded.
```
Comment (glogiotatidis) [inline kitsune/settings.py]: GA_KEY is binary. In SCL3 we have in `settings_local.py` which worked OK but we can't put it as is in an env variable. Base64 encoding it does the trick

File: kitsune/settings.py [hunk 2]
```diff
@@ -1088,11 +1088,10 @@ def JINJA_CONFIG():
 DMS_PROCESS_EXIT_SURVEYS = config('DMS_PROCESS_EXIT_SURVEYS', default=None)
 DMS_SURVEY_RECENT_ASKERS = config('DMS_SURVEY_RECENT_ASKERS', default=None)
 DMS_CLEAR_EXPIRED_AUTH_TOKENS = config('DMS_CLEAR_EXPIRED_AUTH_TOKENS', default=None)
-DMS_UPDATE_VISITORS_METRIC = config('DMS_UPDATE_VISITORS_METRIC', default=None)
+# DMS_UPDATE_VISITORS_METRIC = config('DMS_UPDATE_VISITORS_METRIC', default=None)
```
Comment (glogiotatidis) [inline kitsune/settings.py]: See also #3039

File: bin/run-celery-worker.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec newrelic-admin run-program python manage.py celeryd --maxtasksperchild=${CELERY_MAX_TASKS_PER_CHILD:-10}
```
[the remark this answers is not in the document]
Comment (pmac) [reply bin/run-celery-worker.sh]: 10 does seem pretty low. These tasks are things like updating the search indexes and processing uploaded images. Maybe something like 100 would be more reasonable?

File: kitsune/settings.py (hunk 1 above)
Comment (metadave) [reply kitsune/settings.py]: just a heads up, K8s secrets are base64-encoded, so this value will be double base64 encoded. I don't believe there is any impact here, just wanted to note.

```
echo "foo" | base64 | base64 | base64 -D | base64 -D
```

File: bin/run-celery-worker.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (glogiotatidis) [reply bin/run-celery-worker.sh]: OK, changed default to 100

**Pull request merged**